easybuild/framework/easyblock.py

```diff
@@ -1444,8 +1444,11 @@ def extensions_step(self, fetch=False):
         self.clean_up_fake_module(fake_mod_data)
 
     def package_step(self):
-        """Package software (e.g. into an RPM)."""
-        pass
+        """Prepare package software (e.g. into an RPM) with fpm."""
+
+        path_to_module_file = os.path.join(install_path('mod'), build_option('suffix_modules_path'), self.full_mod_name)
+
+        package_fpm(self, path_to_module_file)
```

Comment:
this should be steered via the added build option

the `easybuild.tools.package` module should probably have a function named `package_with`, that gets passed a string (the value for `build_option(...)`), which tens pick the right `package_foo` function to package with